src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -51,7 +51,7 @@ internal class RpcFunctionInvocationDispatcher : IFunctionInvocationDispatcher
         private ConcurrentStack<WorkerErrorEvent> _languageWorkerErrors = new ConcurrentStack<WorkerErrorEvent>();
         private CancellationTokenSource _processStartCancellationToken = new CancellationTokenSource();
         private CancellationTokenSource _disposeToken = new CancellationTokenSource();
-        private int _debounceMilliSeconds = (int)TimeSpan.FromSeconds(10).TotalMilliseconds;
+        private int _processStartupInterval = (int)TimeSpan.FromSeconds(10).TotalMilliseconds;
```

Comment:
Just make this constant with the proper comment/calculation.

Reply:
can't make this constant as we set the value in workerConfig. I removed the assignment as it's calculated on InitializeAsync call.